### PR TITLE
Adding option to collapse panels in settings

### PR DESCRIPTION
**Pull request by robinjon**
Hi,

I was going to add a different feature, but there was not space in the settings menu. So i added the option to collapse panels. I'm planing to add scrolling and scheduling, not sure if you want to wait util those feature are ready, or if you want to merge now.

### CODE REVIEW

**Comment by robinjon** [comment]
I have tested this on:

Mac OSX High Sierra, Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
Ubuntu 18, openjdk 10.0.2
Windows 10, Java 8

JRE

**Comment by laurent-clouet** [comment]
Thanks for the pull request, i'll test it soon.

Be aware that CollapsibleJPanel.java doesn't follow the code formating of the project, you have its diffinition on code_style_formator.xml

**Comment by robinjon** [comment]
Hope that fixed it.

File: src/com/sheepit/client/standalone/swing/components/CollapsibleJPanel.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
+
```
Comment (intrigus) [inline src/com/sheepit/client/standalone/swing/components/CollapsibleJPanel.java]: This is missing the license header.

**Comment by laurent-clouet** [comment]
Thanks for the PR, it has been merged with 7a2bcf648f807746701bd159866e9912b9129b6d

**Pull request closed without merging**